plans/software/job_hunt_tools.markdown

```diff
@@ -0,0 +1,118 @@
+# Critical Challenges Planning
+
+## The Challenge
+
+* What Critical Challenge are you going to work on?
+
+We need to augment the job hunting process with effective existing tools (both internal and external) and train staff and students to use them.
+
+* Why is this a “challenge”?
+
+Students continue to struggle with surfacing job and networking
+opportunities. But we know there are tons of these out there that we
+haven't even identified.
+
+* Can it be solved? What would be value to the organization/team in solving it or working towards a solution?
+
+Possibly. Value would be more efficiency and coverage in the job-hunt
+process. Possibly removing some of the angst/paralysis students seem to
+encounter in early job-hunt phases that results in lots of inefficient
+time usage.
+
+* What happens if we fail to solve it / improve it?
+
+Students continue pulling from a smaller pool of potential
+opportunities, and we end up having lots of duplicated conversations and
+lost opportunities due to lack of information centralization.
+
+* What skills do you have that make you the right person to work on this challenge?
+
+I've been working on people.turing.io and the turingapps site which are
+targeted at presenting info to employers.
+
+## The Past
+
+* What’s been done so far?
+
+- Job Hunt repo / channel
+- people.turing.io
+- Jeff's manual tracking of student job requirements (resumes,
+  portfolio submissions)
+
+* Was it effective? How do you know?
+
+Definitely better than nothing. Jeff seems to value the portfolio sites
+as a very easy way to share students with employers. Helps us to "strike
+while the iron is hot"
+
+* Was this someone’s challenge before you? Why did they stop doing it or how did we make progress without addressing it?
+
+Jeff mostly.
+
+## The Present
+
+* What are the first steps that you’re ready to execute now?
+
+Need to have some discussions with other Turing staff to try to
+understand what tools are most needed.
+
+* What will be the next three things you do?
+
+1. Investigate previous turing community projects around job stuff (are
+   any of these useful)
+2. Research existing off-the-shelf tools for this type of role to see if
+   any of them would be a good fit for us.
+3. ???
+
+* How will you keep track of progress and results?
+
+Launch a tool and track its adoption. Do we get students to start using
+it and is it helping them.
+
+* Are you empowered to take the actions you think necessary?
+
+## The Future
+
+* What would total success look like?
+
+We would actually know what's going on with job hunt process. Students
+stepping into job hunt process would feel like they were moving into a
+well-controlled and monitored funnel (rather than an amorphous and very
```

Comment:
Do we have any documentation/resources that we give students about the job hunt? A checklist, steps, etc? I know it sounds elementary, but I think that this could make it seem more well-controlled and get rid of some student fears.